src/zenml/orchestrators/step_launcher.py

```diff
@@ -259,6 +331,12 @@ def _bypass() -> None:
                                 force_write_logs=force_write_logs,
                             )
                             break
+                        except (
+                            RunStoppedException,
+                            RunInterruptedException,
+                            KeyboardInterrupt,
+                        ) as e:
+                            raise e
```

Comment:
If the step is interrupted while the pipeline run is still RUNNING (so not because the pipeline run itself was stopped), should we still retry the step here ?